config/locales/views/feedbacks/en.yml

```
@@ -59,12 +59,8 @@ en:
     submissions_table:
       update-submission: "Change to this submission"
       confirm: "Are you sure? All comments on the previous submission will be deleted."
-    score_table:
+    score_card:
       total_score: Total grade
-      score_breakdown: Grade breakdown
-    score_link:
-      score: "Grade: %{score} / %{max}"
-      score_short: "%{score} / %{max}"
-      present: "There are %{count} score items"
-      view_breakdown: View grade breakdown
+      score: "Grade"
       view_feedback: Go to the feedback
+      info_html: "This solution has been evaluated by a teacher. They provided the following feedback. <a>See all evaluated solutions</a>."
```

```suggestion
      info_html: "This solution has been manually evaluated by a teacher as part of <a> an evaluation</a>. Below, you can find your feedback."
```
I think the href/url is missing here.